erpnext/education/doctype/instructor/instructor.js

```diff
@@ -51,3 +50,8 @@ frappe.ui.form.on("Instructor", {
 		}
 	}
 });
+cur_frm.fields_dict['employee'].get_query = function(doc) {
```

Comment:
@sharmatripti Please use `frm.set_query` instead of the cur_frm which is the old way of filtering. Its a simple filter and can be easily implemented using `set_query`.

Reply:
ok